### PR TITLE
Add Chessground boards to broadcast round

### PR DESCRIPTION
**Pull request by TBestLittleHelper**
The board will update when the "refresh" button is used. I think it's worth thinking about making the refresh happen automatically.

The styling can also be improved. The " g " coordinate goes below the board for example. 

This also notably adds [vue3-chessboard](https://github.com/qwerty084/vue3-chessboard) as a dependency.

### CODE REVIEW

File: src/components/RoundBoard.vue [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+});
+</script>
+
+<template>
+  {{ game.name }}
+  <span class="float-right font-bold" :class="{ 'text-red-600': game.status === '*' }">
+    <span v-if="game.status === '*'">LIVE</span> <span v-else>{{ game.status }}</span>
+  </span>
+  <span v-if="showGames">
+    <TheChessboard :boardConfig="boardConfig" reactive-config class="w-auto" />
```
Comment (TBestLittleHelper) [inline src/components/RoundBoard.vue:30]: Using w-auto here, since the default is 90vh, and that overflows everything.

**Comment by TBestLittleHelper** [comment]
This looks ok now, and after using v-show the "lag spike" when toggling boards is gone too. 

User experience wise, I think one would expect the boards to update in real time. Especially if you have a big broadcast and are scrolled way down, the " refresh " button is out of sight. But streaming the broadcast api should probably be a separate PR. It might be worth holding off merging until we do that.

**Comment by fitztrev** [comment]
Looks great!

Ok, so incorporate the PGN stream endpoint into the Round view to make updates to a board?

**Comment by TBestLittleHelper** [comment]
Yep, exactly. If you have time, please do feel free to go for it. Otherwise, I will try to pick it up closer to the end of the summer.

**Comment by fitztrev** [comment]
Ok sounds good. Let's merge it for now, since I'm doing some other UI work and don't want to cause conflicts. We can decide shortly before the next release if we temporarily hide this feature until the stream endpoint has been incorporated.

Really like it. Thanks for adding!

**Pull request merged**